samza-core/src/main/scala/org/apache/samza/coordinator/JobModelManager.scala

```diff
@@ -23,34 +23,22 @@ import java.util
 import java.util.concurrent.atomic.AtomicReference
 
 import org.apache.samza.{Partition, SamzaException}
-import org.apache.samza.config._
```

Comment:
Seems all the changes in this file is just to reorg imports. If that's the case, I think this file can be left untouched.

Reply:
The RB has been updated and aligned with the master branch. The change here has gone.